README.md

```diff
@@ -71,25 +78,41 @@ Import KAMI into your IDE of choice. If you use IntelliJ, import from the `build
 If you do not wish to run from an IDE, use `gradlew.bat runClient` to run KAMI.
 
 ### Building
-#### Windows
-You can build by running `autobuild.bat` and then `autocopy.bat`, or if you prefer the manual way, do the following:
-
+#### Linux
+You can build by running these commands (without the <>) in a terminal with the current directory being KAMI. (EG. `cd ~/Downloads/KAMI`)
 ```
-gradlew.bat build
-cd build/libs
+chmod +x gradlew
+./gradlew <args>
 ```
-To copy the release  jar in `build/libs` you can run `autocopy.bat` or, if you prefer doing it manually, find a file `KAMI-<minecraftVersion>-<kamiVersion>-release.jar` which you can copy to the `mods` folder of a minecraft instance that has forge installed.
-#### Linux
+Possible arguments:
 ```
-git clone https://github.com/zeroeightysix/KAMI.git
-cd KAMI
```

Comment:
These could've stayed.

Reply:
I'm not sure where it disappeared to? Seems to be fine
[Screenshot_20191021_170729](https://user-images.githubusercontent.com/17222512/67243287-46c82580-f425-11e9-869c-e9409f28b6c0.png)

How about this here?
![Screenshot_20191021_170809](https://user-images.githubusercontent.com/17222512/67243347-5d6e7c80-f425-11e9-8f96-a051e4fe9e7f.png)